### ISSUE TITLE
Branding styling with React implementation

### ISSUE THREAD

**Issue by phale98** [Microsoft/BotFramework-WebChat#1779]
I'm attempting to use the branding styling while hosting the application with react

```
import { DirectLine } from 'botframework-directlinejs';
import React from 'react';
import ReactWebChat from 'botframework-webchat';

export default class extends React.Component {
    constructor(props) {
      super(props);
  
      this.directLine = new DirectLine({ token: "XXXXXXXX" });
    }

    styleOption = {
      botAvatarInitials: 'BN',
      userAvatarInitials: 'PH'
    }
    
    render() {
      return (
        <ReactWebChat styleSetOptions={this.styleOption} directLine={ this.directLine } />
      );
    }
  }
}
```

But the website produced when I start the development server does not include the initials in the styleOption. 

![botchat](https://user-images.githubusercontent.com/26072528/53749295-b1aff680-3e6c-11e9-8a82-c97818f22a52.png)

Is this intentional, a feature that just hasn't been added, or am I doing something incorrectly?
Thank you.

**Comment by benbrown** [comment]
I believe the issue may be the parameter name - pass in `styleOptions` rather than `styleSetOptions`

**Comment by phale98** [comment]
I just tried and this did not add the initials either.

**Comment by phale98** [comment]
Seems to be fixed with 4.3.x
